lib/model.js

```diff
@@ -1814,7 +1814,13 @@ class Model {
 
     if (!results.length) return original;
 
-    return Promise.map(options.include, include => {
+    // Filter attribute aliases with the same name
+    const includes = options.attributes?
```

Comment:
This adds additional complexity for something that can be fixed by changing alias name for attributes by user, I don't think this should be fixed. Your thoughts

Reply:
Since the DBMS supports this naming, I expect sequelize to support it as well. The current behavior is unexpected and makes it take time to understand why the ORM is not working as expected.